pkg-r/R/data_source.R

```diff
@@ -21,7 +21,7 @@ as_querychat_data_source.data.frame <- function(x, table_name, ...) {
     length(table_name) == 1 &&
     grepl("^[a-zA-Z][a-zA-Z0-9_]*$", table_name, perl = TRUE)
   if (!is_table_name_ok) {
-    rlang::abort(
+    cli::cli_abort(
       "`table_name` argument must be a string containing alphanumeric characters and underscores, starting with a letter."
     )
```

Comment:
Um... part of wanting to use `cli_abort()` over `rlang::abort()` is to take advantage of cli's inline markup formatting